internal/worker/rundynamic.go

```diff
@@ -120,6 +124,37 @@ func retrieveExecutionLog(ctx context.Context, sb sandbox.Sandbox) (string, erro
 	return string(processedLog), nil
 }
 
+func addSSHKeysToSandbox(sb sandbox.Sandbox, ctx context.Context) error {
```

Comment:
If context is present in an argument list, it is always first (see https://google.github.io/styleguide/go/decisions#contexts)

Reply:
Done.